tests/testrefs.py

```diff
@@ -198,3 +199,31 @@ async def gen_strings2():
     assert task1.done()
     assert not task2.done()
     assert p._param__private.async_refs['string'] is task2
+
+async def test_generator_ref_cancelled():
```

Comment:
```suggestion
@pytest.mark.asyncio
async def test_generator_ref_cancelled():
```

Reply:
We have a ton of asyncio tests that aren't marked as such.

Reply:
We already configure `asyncio_mode = "auto"` in the pyproject.toml, so really don't think this is needed.